File: Examples/CSharp/Articles/RemoveUnusedStyles.cs
using System;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    class RemoveUnusedStyles
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Load template excel file containing unused styles
            Workbook workbook = new Workbook(sourceDir + "sampleRemoveUnusedStyles.xlsx");

            // Remove all unused styles inside the template this will also remove AsposeStyle which is an unused style inside the template
            workbook.RemoveUnusedStyles();

            // Save the file
            workbook.Save(outputDir + "outputRemoveUnusedStyles.xlsx");

            Console.WriteLine("RemoveUnusedStyles executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/RemoveUnusedStyles.vb
Namespace Articles
    Public Class RemoveUnusedStyles
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Load template excel file containing unused styles
            Dim workbook As New Workbook(dataDir & Convert.ToString("Template-With-Unused-Custom-Style.xlsx"))

            ' Remove all unused styles inside the template this will also remove AsposeStyle which is an unused style inside the template
            workbook.RemoveUnusedStyles()

            ' Save the file
            workbook.Save(dataDir & Convert.ToString("output_out_.xlsx"))
            ' ExEnd:1
        End Sub
    End Class
End Namespace